doc/profiling.rst

```diff
@@ -198,6 +198,11 @@ As an example::
     newschedule, _ = p_trans.apply(schedule.children[1:3])
     newschedule.view()
 
+.. warning::
+
+    It is the responsibility of the user to make sure that a profile
```

Comment:
Great. Please could you also extend section 14.2 (Profiling Command Line Options) to mention the ability/need to force PSyclone to do auto-profiling if an optimistion script is also supplied.